acceptance/openstack/sdrs/v1/protectedinstances_test.go

```diff
@@ -17,10 +17,7 @@ func TestSDRSInstanceList(t *testing.T) {
 	th.AssertNoErr(t, err)
```

Comment:
Did you run tests, please attach to description tests result

Reply:
Done